scripts/bumpver.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/python3
+import argparse
+import subprocess
+import sys
+
+
+NAME = "logdetective"
+
+
+def bump_version(filename, target_version=None):
+    """Bump pyproject.toml version."""
+    # Load pyproject.toml file
+    with open(filename, 'r') as f:
+        lines = f.readlines()
+
+    # Find and update version line
+    version_line_index = None
+    for i, line in enumerate(lines):
+        if line.strip().startswith("version ="):
+            version_line_index = i
+            break
+
+    if version_line_index is None:
+        print("Version line not found in pyproject.toml")
+        return
+
+    version_line = lines[version_line_index].strip()
+    version_prefix, version = version_line.split("=")
+
+    if target_version is None:
+        target_version = bump_patch_version(version.strip().strip('"'))
+
+    # Update version in the lines
+    lines[version_line_index] = f'{version_prefix}= "{target_version}"\n'
+
+    # Write back to the file
+    with open(filename, 'w') as f:
+        f.writelines(lines)
```

Comment:
I think you can avoid this logic entirely by using https://docs.python.org/3/library/tomllib.html Since we are on the 3.11 the library is now in builtins.

Reply:
That was my first version but additional library didn't really simplify the code, so I went without it.

However, I missed that,there is new buildin solution.

Reply:
I tried your recommended library and nice improvement. Thanks for recommendation.